### ISSUE TITLE
After release gnome 3.26.1 budgie desktop not start

### ISSUE THREAD

**Issue by Ste74** [budgie-desktop/budgie-desktop#1190]
**Operating System**: Manjaro
**Budgie version**: 10.4

**The issue encountered**
budgie not start but revert on gnome this happen with gnome 3.26.1 
Any idea @ikeydoherty

**Comment by ikeydoherty** [comment]
Lot of Arch people complaining about this. The Arch Budgie package is not properly maintained, and I don't know what package you're using in Manjaro - need details and logs please.

**Comment by olback** [comment]
Yeah, stopped working for me as well. I'd love to send the logs your way if only i knew where to look.

Distro: Antergos (Arch based)
Budgie Desktop v10.3.1-2 wont work at all. When logging in the screen just goes black and I get thrown back to the login screen.

Budgie Desktop v10.4 works a little bit better. After opening and closing one application the background disappears and comes back. Opening and closing an application one more time crashes Budgie and i have to login again.

**Comment by tam1m** [comment]
You need to compile budgie-desktop against libmutter-1. The repo version does not work with the new libmutter after the gnome 3.26.1 update

**Comment by ArtyomFR** [comment]
Same here:
package budgie-desktop 10.3.1-2: 
instant crash and return to login
package budgie-desktop-git v10.4.r6.g9e2cc0f2-1: 
Can login but closing maximized windows make it crash (sometime it arrive when opening).
Beside the crash their is cool new features...

**Comment by ikeydoherty** [comment]
What if you disable animations..?

**Comment by olback** [comment]
@ikeydoherty Did not help, are there logfiles we can provide?

**Comment by Ste74** [comment]
@tam1m I can re push mutter 3.26 instead 3.26.1 if this help

**Comment by philmmanjaro** [comment]
Maximized window issue was introduced with mutter/gnome-shell 3.26.1. So we have to see which commit created that mess. @Ste74: simply downgrading a package makes no sense. Better do a **git-bisect** to find the commit.

**Comment by Ste74** [comment]
Ok i can manage with this this evening ;)

**Comment by branchard** [comment]
Same issue with LightDM:
Distro: Arch
Budgie Desktop 10.3.1-2
Display manager: LightDM 1:1.22.0-1

**Comment by tracktraps** [comment]
https://bugs.archlinux.org/task/55906
I reported the bug yesterday. It's a shame that Budgie is not properly maintained on Arch Linux :(

**Comment by branchard** [comment]
So, do you have a solution for now ?
I had to install Mate Desktop to work this morning. What a waste of time ...

**Comment by Ste74** [comment]
> If you turnoff animations via Tweaks, then crashing whilst maximised/tiled does not occur (or at least much harder to invoke a crash).

Temporary solution waiting finding the regression

**Comment by JeanBarriere** [comment]
Hi !
Same issue at first, but when i disabled the animations, it launches now with gnome topbar.
It does not crash but budgie does not launch.

Distribution: ArchLinux
Budgie Desktop 10.3.1-2
Display manager: LightDM 1:1.22.0-1

**Comment by branchard** [comment]
Did you try with Budgie AUR?

**Comment by JeanBarriere** [comment]
@branchard Yes I tried but it does not compile anymore:

```
Project name: gvc
Native C compiler: cc (gcc 7.2.0)
Native dependency libpulse found: YES 11.1
Native dependency libpulse-mainloop-glib found: YES 11.1

Meson encountered an error in file subprojects/gvc/meson.build, line 31, column 0:
Sandbox violation: Tried to grab file gvc-channel-map.c from a different subproject.

A full log can be found at ....../src/budgie-desktop-git/build/meson-logs/meson-log.txt
==> ERROR: A failure occurred in build().
    Aborting...
```

**Comment by ikeydoherty** [comment]
Kinda looks like you're doing something janky with the packaging and not using our release tarballs

**Comment by JeanBarriere** [comment]
I use the following command : `yaourt -S budgie-desktop-git`
So I'm not doing anything weird with the package.

**Comment by ArtyomFR** [comment]
> Same issue at first, but when i disabled the animations, it launches now with gnome topbar.

Same issue as me I think. We see budgie trying to start but crash directly.

>  Yes I tried but it does not compile anymore

I manage to compile it with meson from the 29th september.

**Comment by JeanBarriere** [comment]
I followed Brad Ford instructions on this bug https://bugs.archlinux.org/task/55906

> 
> HOWEVER, if you have recently updated meson (meson-0.43.0-1), the build will fail with the error:
>
> \# Meson encountered an error in file subprojects/gvc/meson.build, line 31, column 0:
> \# Sandbox violation: Tried to grab file gvc-channel-map.c from a different subproject.
>
> If this happens, you will need to downgrade meson to meson-0.42.1-2 (or older). If you are lucky, you may have the old package in your package cache:
>
> \# pacman -U /var/cache/pacman/pkg/meson-0.43.0-1-any.pkg.tar.xz

So i downgrade meson to version `0.42.1-3` and retried to install `budgie-desktop-git` version : it works.

**Comment by santisteban** [comment]
I can confirm that Brad instructions worked for me too.

After downgrading `meson` to `meson-0.42.1-3` I uninstalled `budgie-desktop` with `pacman` (using `-R`), then installed `budgie-desktop-git` with `pacaur` (using `-S`).

I also have animations disabled and I have yet to experience a crash.

Arch Linux x86_64
budgie-desktop v10.4-6-g9e2cc0f2

**Comment by alunux** [comment]
@JeanBarriere @santisteban You can build the latest master branch using meson 0.43.0, now.

**Comment by Ste74** [comment]
Solved with mutter 3.26.1-2
